pkg/network/infraconfigurators/bridge.go

```diff
@@ -73,10 +73,6 @@ func (b *BridgePodNetworkConfigurator) DiscoverPodNetworkInterface(podIfaceName
 		b.vmMac = &b.podNicLink.Attrs().HardwareAddr
```

Comment:
Do you mind add the PR desc also to the commit message? 
So the "why" part will appear in the commit history

Reply:
I thought that was obvious.

Done.